Make install issue's

Looks like when I try and run a "make install" using the most recent commit I get this:

```
Install the project...
-- Install configuration: "Debug"
-- Installing: /usr/local/bin/servatrice
-- Removed runtime path from "/usr/local/bin/servatrice"
-- Installing: /usr/local/share/servatrice/servatrice.ini.example
-- Installing: /usr/local/share/servatrice/servatrice.sql
-- Installing: /usr/local/share/icons/hicolor/48x48/apps/servatrice.png
-- Installing: /usr/local/share/icons/hicolor/scalable/apps/servatrice.svg
-- Installing: /usr/local/share/applications/servatrice.desktop
-- Installing: /usr/local/bin/cockatrice
-- Removed runtime path from "/usr/local/bin/cockatrice"
-- Installing: /usr/local/share/icons/hicolor/48x48/apps/cockatrice.png
-- Installing: /usr/local/share/icons/hicolor/scalable/apps/cockatrice.svg
-- Installing: /usr/local/share/applications/cockatrice.desktop
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_nb.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_pl.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_sv.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_zh-Hans.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_en.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_cs.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_fr.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_de.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_ja.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_en@pirate.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_uk.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_pt.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_es.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_pt_BR.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_ko.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_it.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_et.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_ru.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_zh-Hant.qm
-- Installing: /usr/local/share/cockatrice/translations/cockatrice_nl.qm
-- Installing: /usr/local/share/cockatrice/sounds/attack.raw
-- Installing: /usr/local/share/cockatrice/sounds/tap.raw
-- Installing: /usr/local/share/cockatrice/sounds/start_combat.raw
-- Installing: /usr/local/share/cockatrice/sounds/end_step.raw
-- Installing: /usr/local/share/cockatrice/sounds/player_joined.raw
CMake Error at /home/wcollins/Github/Cockatrice/build/themes/cmake_install.cmake:36 (FILE):
  file INSTALL cannot find "/home/wcollins/Github/Cockatrice/themes/Default".
Call Stack (most recent call first):
  cmake_install.cmake:41 (INCLUDE)


make: *** [install] Error 1
```

@ctrlaltca  Is there a folder not being generated that needs to be?

--- a/themes/Default/.gitignore
+++ b/themes/Default/.gitignore
@@ -1,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore

